recipe/build.sh

```diff
@@ -16,9 +16,9 @@ mkdir build_lib
 cd build_lib
 cmake -D BUILD_LIB=ON -D BUILD_SHARED_LIBS=ON -D BUILD_MPI=OFF -D BUILD_OMP=OFF $args ../cmake
 make # -j${NUM_CPUS}
-cp liblammps.* ../src  # For compatibility with the original make system.
-cd ../python
-python install.py 
+cp liblammps{.dylib,.so} ../src  # For compatibility with the original make system.
```

Comment:
use ${SHLIB_EXT}
```suggestion
cp liblammps${SHLIB_EXT} ../src  # For compatibility with the original make system.
```

Reply:
I agree - that is more for debugging at the moment - I change it once the test pass